src/components/cards/ListViewTaskCard.tsx

```diff
@@ -102,28 +102,18 @@ export const ListViewTaskCard = ({
                 getSelectedValue={(_newValue) => {
                   const newValue = _newValue as IAssigneeCombined
                   updateAssigneeValue(newValue)
-                  const assigneeType = newValue?.type
+                  const assigneeType = newValue.type as AssigneeType
```

Comment:
Are we sure that there is no case where `newValue` can be undefined / null here?

Reply:
in IAssigneeCombined :  type: AssigneeType, I think type can never be undefined for an assignee value.

Reply:
On second thoughts, there should be no assignee whose value will be null in the future, so I think it should be handled. Thanks for pointing that out